sprout/appliances/tasks.py

```diff
@@ -1968,8 +1971,8 @@ def process_docker_images_from_url_group(self, group_id, version, docker_version
             continue
         try:
             Template.objects.get(
-                ~Q(container=None), template_group=group, provider=provider, version=version,
-                date=date, preconfigured=True)
+                ~Q(container=None) & ~Q(provider__provider_type='openshift'), template_group=group,
+                provider=provider, version=version, date=date, preconfigured=True)
```

Comment:
I think this should just plainly skip and not execute at all for openshift.

Reply:
you're right. I even added such check several rows before